ecl/eclcc/eclcc.cpp

```diff
@@ -2248,6 +2248,7 @@ const char * const helpText[] = {
     "! -fexportDependencies    Generate information about inter-definition dependencies",
     "! -fmaxCompileThreads     Number of compiler instances to compile the c++",
     "! -fnoteRecordSizeInGraph Add estimates of record sizes to the graph",
+    "! -fobfuscateOutput       Remove details of the original ECL from output",
```

Comment:
I don't think it is significant enough to include.  This list doesn't include all options - only those that are possibly useful.